### ISSUE TITLE
ECONNREFUSED when parsing medium posts

### ISSUE THREAD

**Issue by aterreno** [mathieudutour/medium-to-own-blog#78]
**Describe the bug**
```
✔ Parsed Medium content
✔ Parsed the Medium profile
✖ Parsing the Medium posts [5/215]

Error: connect ECONNREFUSED 198.54.117.197:443
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16) {
  errno: -61,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '198.54.117.197',
  port: 443
}
```

I'd like to know which article is causing the issue and what calls the importer is making, in order to troubleshoot the issue. 

Thanks

**Comment by aterreno** [comment]
I ended up cloning the source code and adding some debug logs when fetching urls. 

The issue is quite peculiar to my own medium profile, where I have a custom domain setup, I did not renew the domain and this tool seems to connect to the medium post anyway, not just getting the data from the zip archive. 

I hope this helps if anyone else would attempt to do the migration in the same state, I've asked their support team to disable my domain now and hopefully I'll be able soon to proceed with the migration.
